fjord/feedback/views.py

```diff
@@ -45,7 +46,17 @@ def download_firefox(request, template):
 
 
 def thanks(request):
-    template = 'feedback/thanks.html'
+    opinion_id = request.session['opinion_id']
+    user_opinion = Response.objects.get(id=opinion_id)
```

Comment:
Calling `.get()` can raise `MultipleObjectsReturned` (though with an id, that's super unlikely--our db would have to be totally fux0rd) and `DoesNotExist` (which is possible). We should handle the `DoesNotExist` scenario.

https://docs.djangoproject.com/en/1.6/ref/models/querysets/#get